packages/runtime/test-runtime-utils/src/mocksForReconnection.ts

```diff
@@ -29,6 +34,10 @@ export class MockContainerRuntimeForReconnection extends MockContainerRuntime {
 
         if (connected) {
             this.clientSequenceNumber = 0;
```

Comment:
Nit: the clientSequenceNumber reset should also probably happen after the processing of remote messages.  Technically in ConnectionManager we reset it after the first submit with a new clientId, but resetting it more-or-less simultaneously with the clientId change is probably close enough.